rest-api-spec/src/main/resources/rest-api-spec/api/delete_all_pits.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
Why didn't we add corresponding test file under `rest-api-spec/src/main/resources/rest-api-spec/test`?

Reply:
Added tests